### ISSUE TITLE
UCAR models are broken

### ISSUE THREAD

**Issue by nwagenbrenner** [firelab/windninja#467]
It looks like there was a file path change.

**Comment by nwagenbrenner** [comment]
I've updated file paths but it appears that functionality of the keyword `present` when defining a time range has changed. Instead of pointing to the present time it points to the latest timestep for the forecast on the server. For example, this request downloads data valid for Jan 26 17:00 MST (the last timestep available on their server), but it is currently Jan 19 13:18 MST. This breaks our downloads because we specify the `start_time` as `present` and download past this with the `time_duration` argument. I just contacted the THREDDS helpdesk and am waiting to hear back.

https://thredds.ucar.edu/thredds/ncss/grid/grib/NCEP/NDFD/NWS/CONUS/NOAAPORT/Best?north=44.4371&west=-114.076&east=-111.98&south=42.3603&time=present&accept=netcdf3&var=Maximum_temperature_height_above_ground_12_Hour_Maximum,Minimum_temperature_height_above_ground_12_Hour_Minimum,Total_cloud_cover_entire_atmosphere_single_layer_layer,Wind_direction_from_which_blowing_height_above_ground,Wind_speed_height_above_ground

**Comment by nwagenbrenner** [comment]
The intended behavior is described here:

https://docs.unidata.ucar.edu/tds/5.0/userguide/ncss_grid.html

**Comment by nwagenbrenner** [comment]
Also note that the behavior of `present` seems inconsistent among the models. For the NAM it appears to download past data starting on Jan 13.

**Comment by nwagenbrenner** [comment]
We just got the following update from THREDDS Support:

```
We were able to resolve this issue and anticipate this change to be included in TDS 5.5 which will be released in a few weeks. Meanwhile, the change is available on our test server at thredds-test.unidata.ucar.edu

Ticket Details
===================
Ticket ID: NJL-799762
Department: Support THREDDS
Priority: High
Status: Closed
===================
```

We should test with the test server. If things look good, this issue should be resolved in the coming weeks without any further work on our side.

**Comment by nwagenbrenner** [comment]
Fixed via THREDDS suppport.
